auth/auth-server/src/error.rs

```diff
@@ -25,6 +25,9 @@ pub enum AuthServerError {
     /// Error setting up the auth server
     #[error("Error setting up the auth server: {0}")]
     Setup(String),
+    /// An error setting up the token remapping
+    #[error("Error setting up the token remapping: {0}")]
+    TokenRemap(String),
```

Comment:
probably makes more sense to just use the `Setup` error for this, no need to define a new enum variant